tools/gen_ort_dockerfile.py

```diff
@@ -121,21 +121,22 @@ def dockerfile_for_linux(output_file):
         df += '''
 # Install OpenVINO
 ARG ONNXRUNTIME_OPENVINO_VERSION
-ENV INTEL_OPENVINO_DIR /opt/intel/openvino_${ONNXRUNTIME_OPENVINO_VERSION}
-ENV LD_LIBRARY_PATH $INTEL_OPENVINO_DIR/deployment_tools/inference_engine/lib/intel64:$INTEL_OPENVINO_DIR/deployment_tools/ngraph/lib:$INTEL_OPENVINO_DIR/deployment_tools/inference_engine/external/tbb/lib:/usr/local/openblas/lib:$LD_LIBRARY_PATH
+ENV INTEL_OPENVINO_DIR /opt/intel/openvino_2022
+ENV LD_LIBRARY_PATH $INTEL_OPENVINO_DIR/tools/compile_tool:$INTEL_OPENVINO_DIR/runtime/3rdparty/tbb/lib:$INTEL_OPENVINO_DIR/runtime/3rdparty/hddl/lib:$INTEL_OPENVINO_DIR/runtime/lib/intel64:/usr/local/openblas/lib:$LD_LIBRARY_PATH
 ENV PYTHONPATH $INTEL_OPENVINO_DIR/tools:$PYTHONPATH
-ENV IE_PLUGINS_PATH $INTEL_OPENVINO_DIR/deployment_tools/inference_engine/lib/intel64
-ENV InferenceEngine_DIR=$INTEL_OPENVINO_DIR/deployment_tools/inference_engine/share
-ENV ngraph_DIR=$INTEL_OPENVINO_DIR/deployment_tools/ngraph/cmake
+ENV IE_PLUGINS_PATH $INTEL_OPENVINO_DIR/runtime/lib/intel64
```

Comment:
I think these names here need an update. Openvino doesn't have inference_engine or ngraph abstraction anymore right?

Reply:
Actually, openvino retained the inference_engine and ngraph, and simply added the openvino api 2.0 with the 2022 release. The new `ov` namespace also have the old `InferenceEngine` on top of it [here](https://github.com/openvinotoolkit/openvino/blob/2022.1.0/src/core/include/openvino/core/any.hpp#L23).
The paths are needed for a successful build.